### PR TITLE
Retry failed schema diff migrations

### PR DESCRIPTION
**Pull request by ausi**

### CODE REVIEW

File: core-bundle/src/Command/MigrateCommand.php [hunk 1]
```diff
@@ -243,13 +243,37 @@ static function ($hash) use ($commandsByHash) {
...
+                        ++$count;
+                        ++$successCount;
+                        unset($commandHashes[$key]);
+                        $this->io->writeln('');
+                    } catch (\Throwable $e) {
+                        $this->io->writeln('......FAILED');
+                        $exceptions[] = $e;
+                    }
+                }
+            } while ($successCount > 0);
```
Comment (m-vo) [inline core-bundle/src/Command/MigrateCommand.php]: I fail to see how this would terminate?

Comment (Toflar) [reply core-bundle/src/Command/MigrateCommand.php]: It will stop, once no command could be executed successfully anymore.

Comment (m-vo) [reply core-bundle/src/Command/MigrateCommand.php]: Ah my bad, I thought the variable initialization was outside the loop. :smile:  
Btw.: If the count isn't used at all I'd prefer a boolean (`while($commandExecuted);`).

Comment (ausi) [reply core-bundle/src/Command/MigrateCommand.php]: Done in aa7439ef3ef331a3b63a774aeb91804319ac81e3

**Review by m-vo — DISMISSED** [review]

**Review by bytehead — DISMISSED** [review]

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @ausi.

**Pull request merged**